ui/src/containers/Header/Header.test.tsx

```diff
@@ -12,6 +12,16 @@ const TESTDATA_DIR = `src/store/testdata`;
 const api = new FakeHub(TESTDATA_DIR);
 const { Provider, root } = createProviderAndStore(api);
 
+// Assign a dummy date to global Date inorder to avoid test failue
+beforeEach(() => {
+  global.Date.now = jest.fn(() => new Date('2021-01-01T10:20:30Z').getTime());
```

Comment:
I think we should create a function for this in `common` directory since it is used at many places

Reply:
this is a hook I guess....so not sure whether we can place it in commons directory :thinking:

Reply:
okay will check it once :+1: